go/chat/storage/inbox.go

```diff
@@ -254,6 +254,19 @@ func (i *Inbox) Merge(ctx context.Context, vers chat1.InboxVers, convsIn []chat1
 	return nil
 }
 
+func (i *Inbox) supersedersNotEmpty(ctx context.Context, superseders []chat1.ConversationMetadata, convs []chat1.Conversation) bool {
+	for _, superseder := range superseders {
+		for _, conv := range convs {
+			if superseder.ConversationID.Eq(conv.Metadata.ConversationID) {
+				if len(conv.MaxMsgs) > 0 {
```

Comment:
Are you sure your test works? Every conversation has at least one max message, so I am surprised that this check works. It should do something similar to Gregor where it checks to see if there are any text or attachment messages in the thread.